api/renter.go

```diff
@@ -29,7 +31,6 @@ type FileInfo struct {
 	UploadProgress float32
 	Nickname       string
 	Filesize       uint64
-	Repairing      bool
```

Comment:
I'm pretty sure it was in use for 0.3.3.3. I definitely wrote and rewrote some repairing code at some point. I also remember it never worked very well.